src/common/model/ITeamChannel copy.ts

```diff
@@ -0,0 +1,9 @@
+export interface ITeamChannel {
```

Comment:
Please, rename the file to remove `copy`

Reply:
changed

Reply:
@joaojmendes the file is still named "ITeamChanell copy.ts" is it being used? If used, can you rename to "ITeamChanell.ts", if not, can you delete?

Reply:
removed file